src/Microsoft.SqlTools.ServiceLayer/EditData/UpdateManagement/RowEdit.cs

```diff
@@ -58,8 +60,31 @@ protected RowEditBase(long rowId, ResultSet associatedResultSet, IEditTableMetad
         /// </summary>
         public IEditTableMetadata AssociatedObjectMetadata { get; }
 
+        /// <summary>
+        /// Sort ID for a row edit. Ensures that when a collection of RowEditBase objects are
+        /// sorted, the appropriate types are sorted to the top.
+        /// </summary>
+        protected abstract int SortId { get; }
```

Comment:
Since the rest of this file is so organized, you might as well stick this in Abstract Methods and rename it to something like "Abstract" or "Abstract Entities".